husky_components/tabs/main.js

```diff
@@ -50,6 +50,7 @@ define(function() {
             forceReload: false,
             callback: null,
             forceSelect: true,
+            conditionData: {},
```

Comment:
Moving the discussion from https://github.com/sulu-io/sulu/pull/2099#discussion_r55395972 to here. What about renaming it to `formData`? And if I am right with https://github.com/massiveart/husky/pull/614/files#r56297942 we could also theoretical rename it do `data`, although having `this.data` and `data` is also anything else than optimal :confused: